yb-voyager/cmd/export.go

```diff
@@ -296,10 +297,10 @@ func validatePortRange() {
 }
 
 func validateSSLMode() {
-	if source.DBType == ORACLE || slices.Contains(validSSLModes[source.DBType], source.SSLMode) {
+	if source.DBType == ORACLE || slices.Contains(ValidSSLModesForSourceDB[source.DBType], source.SSLMode) {
```

Comment:
I'm confused by this existing condition. `If source = oracle OR sslmode is valid` ? for oracle, we don't check the valid ssl modes then?

Ah, I remember; this is because in oracle, we don't really support ssl modes, we rely on TNS ALIAS. Can you just add a comment here?